Assembly scanner docs for azure functions (#5426)
Particular/docs.particular.net: nservicebus/hosting/azure-functions-service-bus/index_assembly-scanner_ASBFunctions_[2.0,).partial.md

### Assembly scanning

The assembly scanner loads assemblies from two locations:

- The `bin` directory of the function application
- The Azure Function runtime directory 

> Loading an assembly the second time (for example from a different directory), would cause runtime exceptions. Contact [Particular support](https://particular.net/support) for assistance. 

Reviewer: ```suggestion
Loading the same assembly from two different locations will cause runtime exceptions. Contact [Particular support](https://particular.net/support) for assistance. 
```
